apps/mobile/src/features/ImportReadOnly/components/ImportAccountFormView.tsx

```diff
@@ -91,6 +91,7 @@ export const ImportAccountFormView: React.FC<ImportAccountFormViewProps> = ({
         onPress={onContinue}
         disabled={!isEnteredAddressValid || !safeExists}
         testID={'continue-button'}
+        marginBottom={70}
```

Comment:
Thanks for the contribution @chiranjeev13 . However I think a more concise approach would be better here, we need to understand what is causing the button to go way down the screen instead of just adding a big margin bottom to it. Might be due to the usage of this hook `useSafeAreaPaddingBottom` in our component.